### PR TITLE
docs: update workspace docs

### PR DESCRIPTION
**Pull request by dammy001**

### CODE REVIEW

File: packages/ui/client/composables/client/static.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type { WebSocketHandlers } from 'vitest/src/api/types'
 import { parse } from 'flatted'
 import { decompressSync, strFromU8 } from 'fflate'
 import type { File, ModuleGraphData, ResolvedConfig } from 'vitest/src/types'
-import { StateManager } from '../../../../vitest/src/node/state'
```
Comment (sheremet-va) [inline packages/ui/client/composables/client/static.ts:7]: How is this related to documentation?

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: packages/ui/client/composables/client/static.ts (hunk 1 above)
Comment (dammy001) [reply packages/ui/client/composables/client/static.ts:7]: Mistake. i'll revert this and open a new PR for it.

**Comment by dammy001** [comment]
@sheremet-va updated.

**Review by sheremet-va — APPROVED** [review]

**Comment by sheremet-va** [comment]
> @sheremet-va updated.

Thank you!
